packages/graphql/src/translate/create-auth-and-params.ts

```diff
@@ -24,6 +24,11 @@ import type { AuthOperations } from "../types/deprecated/auth/auth-operations";
 import type { Rule } from "./deprecated/auth/types";
 import { createAuthPredicates } from "./create-auth-predicates";
 
+type AuthAndParams = {
+    cypher: string;
+    params: Record<string, any>;
```

Comment:
If at all possible I think it would be really nice to narrow down this type if only a little. Perhaps it's not possible at the moment but I think replacing the `any` here would be a good move.

Reply:
This is purely a compatibility fix which is going to help me build out the new authorization translation - it's just a one to one map of the existing signature. This function will be deleted within the next few months anyway, so it's not worth massive time investment.